pyproject.toml

```diff
@@ -95,7 +99,8 @@ addopts = """
     --ignore=src/blueapi/startup
     """
 # https://iscinumpy.gitlab.io/post/bound-version-constraints/#watch-for-warnings
-filterwarnings = ["error", "ignore::DeprecationWarning"]
+# Unignore UserWarning after Pydantic warning removed from bluesky/bluesky and release
+filterwarnings = ["error", "ignore::DeprecationWarning", "ignore::UserWarning"]
```

Comment:
Is there any way to only ignore warnings from the tests in question? Otherwise we may end up removing this in a few months time only to find the tests blowing up in a bunch of other places?